ports/stm32/boards/stm32l476xe.ld

```diff
@@ -11,6 +11,7 @@ MEMORY
     FLASH_FS (r)    : ORIGIN = 0x08060000, LENGTH = 128K /* sectors 192-255 */
     RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 96K
     SRAM2 (xrw)     : ORIGIN = 0x10000000, LENGTH = 32K
+    FS_CACHE(xrw)   : ORIGIN = 0x10007800, LENGTH = 2K
```

Comment:
This looks like a change to existing functionality, in that it was originally placed at 0x10000000.  But since the cache is only 2k it does indeed make sense to put it at the end of this 32k RAM region.  And you tested it so I agree this is a good change.